src/mito2/src/sst/index/indexer/abort.rs

```diff
@@ -87,4 +89,31 @@ impl Indexer {
             );
         }
     }
+
+    async fn do_abort_clean_fs_temp_dir(&mut self) {
+        let Some(puffin_manager) = &self.puffin_manager else {
+            return;
+        };
+        let fs_accessor = puffin_manager.file_accessor();
+
+        let fs_handle = RegionIndexId::new(
+            RegionFileId::new(self.region_id, self.file_id),
+            self.index_version,
+        );
+        let Err(err) = fs_accessor.clean_by_index_id(&fs_handle).await else {
+            return;
+        };
```

Comment:
The path here may be incorrect, the temp file is under a tmp dir, not the path provided by the path_provider.